SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -401,18 +401,25 @@ Buffer {
 	}
 
 	get { arg index, action;
-		OSCpathResponder(server.addr, ['/b_set', bufnum, index], { arg time, r, msg;
-			action.value(msg.at(3)); r.remove }).add;
-		server.listSendMsg(["/b_get", bufnum, index])
+		OSCFunc({ |message|
+			// The server replies with a message of the form [/b_set, bufnum, index, value].
+			// We want "value," which is at index 3.
+			action.value(message[3]);
+		}, \b_set, server.addr, argTemplate: [bufnum, index]).oneShot;
+		server.listSendMsg(["/b_get", bufnum, index]);
```

Comment:
When you check for `\b_set` (a symbol), it might be simpler to read if you also send `\b_get` instead of `"/b_get"`. But this is a matter of taste, too, or maybe another PR.